packages/manager/modules/pci/src/projects/project/instances/instance/instance.constants.js

```diff
@@ -0,0 +1,21 @@
+export const DEDICATED_IPS_URL = {
+  CA: 'https://ca.ovh.com/manager/#/configuration/ip',
+  EU: 'https://www.ovh.com/manager/dedicated/index.html#/configuration/ip',
+  US: 'https://us.ovhcloud.com/manager/dedicated/#/configuration/ip',
+};
+
+export const MITIGATION_URL = (
```

Comment:
It's not a constant then 🤔

Reply:
Part of it because it depends on the `region` but it feels like it doesn't belong in a service for instance as an `URL` is a constant (event if it has to be generated dynamically). I don't know if that makes sense to you